addons/packages/grafana/7.5.16/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: grafana/grafana:7.5.16
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: 7.5.16
+          url: grafana/grafana:7.5.16
+  image: index.docker.io/grafana/grafana@sha256:9b7b1d9a1deadbe6fed74416d58db47f8af31b5cee214cfb659f89e6dda3f716
```

Comment:
I think we should generate our own image? Refer to here image from v7.5.11: projects.registry.vmware.com/tce/grafana@sha256:ca56e9bc0e249ac006717f57f3fa1fdb3888419b224bfa305a67c28caa1364ba.